src/client/features/interactions/interactions-edge-tooltip.js

```diff
@@ -9,13 +9,22 @@ class InteractionsEdgeTooltip extends React.Component {
   constructor(props){
     super(props);
 
+    const edges = props.edge.parallelEdges();
```

Comment:
Preferably, graph topology calls shouldn't be made at all in `render()` -- because `render()` can be called many times.  Since react can destroy and re-create components, it would be even better to raise the parallel edges call higher up in the component hierarchy.  But that could be considered for some future version; this is good enough for now.